ep/build-velox/src/build_velox.sh

```diff
@@ -65,6 +65,19 @@ for arg in "$@"; do
   esac
 done
 
+function apply_compilation_fixes {
+  current_dir=$1
+  velox_home=$2
+  sudo cp ${current_dir}/modify_velox.patch ${velox_home}/
+  sudo cp ${current_dir}/modify_arrow.patch ${velox_home}/third_party/
```

Comment:
@rui-mo Do we need to git apply modify_arrow.patch like modify_velox.patch in  line 74?

Reply:
It is applied to Arrow during compilation: https://github.com/oap-project/gluten/pull/3520/files#diff-b41402fae332ead78c4646843c9bcf66a2813203cdc82521c9c9f130b48146dbR100.